cfgmgr/buffermgrdyn.cpp

```diff
@@ -939,10 +1097,29 @@ task_process_status BufferMgrDynamic::handleDefaultLossLessBufferParam(KeyOpFiel
             if (fvField(i) == "default_dynamic_th")
             {
                 m_defaultThreshold = fvValue(i);
-                SWSS_LOG_DEBUG("Handling Buffer Maximum value table field default_dynamic_th value %s", m_defaultThreshold.c_str());
+                SWSS_LOG_DEBUG("Handling Buffer parameter table field default_dynamic_th value %s", m_defaultThreshold.c_str());
+            }
+            if (fvField(i) == "over_subscribe_ratio")
+            {
+                newRatio = fvValue(i);
+                SWSS_LOG_DEBUG("Handling Buffer parameter table field over_subscribe_ratio value %s", fvValue(i).c_str());
             }
         }
     }
+    else
+    {
+        SWSS_LOG_ERROR("Unsupported command %s received for DEFAULT_LOSSLESS_BUFFER_PARAMETER table", op.c_str());
+        return task_process_status::task_failed;
+    }
+
+    if (newRatio != m_overSubscribeRatio)
+    {
+        bool isSHPEnabled = isNonZero(m_overSubscribeRatio);
+        bool willSHPBeEnabled = isNonZero(newRatio);
+        m_overSubscribeRatio = newRatio;
+        SWSS_LOG_INFO("Recalculate shared buffer pool size due to over subscribe ratio is updated to %s", m_overSubscribeRatio.c_str());
```

Comment:
include the old ratio and new ratio in the log and update 'm_overSubscribeRatio' after the log